composer.json

```diff
@@ -28,6 +32,15 @@
         }
     },
     "require-dev": {
-        "sllh/php-cs-fixer-styleci-bridge": "^2.1"
+        "matthiasnoback/symfony-config-test": "^3.0",
+        "php-http/guzzle6-adapter": "^1.1",
+        "php-http/message": "^1.5",
+        "sllh/php-cs-fixer-styleci-bridge": "^2.1",
+        "symfony/dependency-injection": "^2.8 || ^3.0",
+        "symfony/http-kernel": "^2.8 || ^3.0"
+    },
+    "suggest": {
+        "php-http/httplug-bundle": "For Symfony integration as a bundle",
+        "symfony/http-kernel": "For Symfony integration as a bundle"
```

Comment:
This line is not needed. If we require httplug-bundle, http-kernel will be required too.